### DISCUSSION TITLE
KeyPress and KeyRelease events support

### DISCUSSION THREAD

**Discussion by pratclot** [streamdeck-linux-gui/streamdeck-linux-gui#206] [Ideas / Feature Requests]
Hey there,

Was wondering if the software can assign separate actions for button press and release events? Right now I see that releasing a physical button is ignored.

Thanks

**Comment by coolapso** [comment]
Hi, as far as remember it does not, however this application will no longer get any updates, please read: https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/discussions/203
